File: Competition/CodeChef/July20B/CRDGAME.cpp
#include <bits/stdc++.h>
using namespace std;
using vi = vector <int> ;
using ll = long long;

#define IOS ios::sync_with_stdio(false); cin.tie(nullptr); cout.tie(nullptr);
#define endl "\n"
#define int long long int


// const int N=1e5+5;

int digCount(int x){
    int sum = x % 10;
    while ((x /= 10) != 0)
        sum += x % 10;
    return sum;
}

int32_t main()
{
    IOS;
    int t;
    cin>>t;
    while(t--){
        int n , morty  = 0 , chef = 0 ;
        cin>>n;
        for(int i = 0 ; i<n ; i++){
            int ch1 , mo1;
            cin>>ch1>>mo1;
            int ch = digCount(ch1);
            int mo = digCount(mo1);
            if(ch == mo){
                chef++;
                morty++;
            }
            else if(ch<mo)
                morty++;
            else
                chef++;
        }
        if(chef > morty){
            cout<<"0 "<<chef<<"\n";
        }
        else if(chef < morty){
            cout<<"1 "<<morty<<"\n";
        }
        else{
            cout<<"2 "<<chef<<"\n";
        }

    }
    return 0;
}

File: Competition/CodeChef/July20B/CRDGAME.py
def digSum(n): 
  
    if (n == 0): 
        return 0
    if (n % 9 == 0): 
        return 9 
    else: 
        return (n % 9) 

for _ in range(int(input())):
    n = int(input())
    chef= 0
    morty = 0
    for _ in range(n):
        ch1 , mo1 = list(map(int,input().split(' ')))
        ch = digSum(ch1)
        mo = digSum(mo1)
        
        if (ch == mo):
            chef+=1
            morty+=1
        elif(ch < mo):
            morty+=1
        else:
            chef+=1
    # print(chef,morty)
    if(chef>morty):
        print("0",chef,sep=" ")
    elif(chef<morty):
        print("1",morty,sep=" ")
    else:
        print("2",morty,sep=" ")